src/utils.ts

```diff
@@ -5,6 +5,22 @@ export interface BoundingBox {
   height: number;
 }
 
+export interface ElementOverlayStyleOptions {
+  background?: string;
+  borderColor?: string;
+  borderStyle?: string;
+  borderRadius?: string;
+  borderWidth?: string;
+  boxSizing?: string;
+  cursor?: string;
+  position?: string;
+  zIndex?: string;
+};
+
+export type ElementOverlayOptions = {
+  className?: string;
+  style?: ElementOverlayStyleOptions;
+};
```

Comment:
Style nit: would you mind adding a blank line between the type declaration and the function below?